File: Pizzawahala.py
sapa_slices=4
small_money_slices=6
big_boys_slices=8
odogwu_slices=12
sapa_prices=2000
small_money_prices=2400
big_boys_prices=3000
odogwu_prices=4200
number_of_people=int(input(""" Welcome to Iya Scambirah Pizza joint!!!!
What is the number of People?\n"""))
pizza_type=(input("""We have:
Sapa Size
Small money
Big Boys
Odogwu
What type of pizza do you want to order Sir/Ma?\n
"""))

pizza_slices=0
pizza_prices=0


if pizza_type.lower() == "sapa size" :
    pizza_slices=sapa_slices
    pizza_prices=sapa_prices
elif pizza_type.lower() == "small money" :
    pizza_slices=small_money_slices
    pizza_prices=small_money_prices
elif pizza_type.lower() == "big boys" :
    pizza_slices=big_boys_slices
    pizza_prices=big_boys_prices
elif pizza_type.lower() == "odogwu" :
    pizza_slices=odogwu_slices
    pizza_prices=odogwu_prices



boxes=1
while boxes * pizza_slices < number_of_people:
    boxes+=1
    pizza_bill=boxes * pizza_prices
    leftover_pizza=boxes * pizza_slices - number_of_people






print("The number of boxes to buy=", boxes , "boxes")
print("Number of left over slices after serving=", leftover_pizza , "slices")
print("Prices", pizza_bill)

File: Pizza wahala/Pizzawahala.java
import java.util.Scanner;
public class Pizzawahala{
public static void main(String[]args){
Scanner input=new Scanner(System.in);
int sapaslice= 4;                             
int smallslice= 6;                                    
int bigslice=8;                                        
int odogwuslice=12;
int sapaprice=2000;
int smallprice=2000;
int bigprice=3000;
int odogwuprice=4200;                         
System.out.println("""

Welcome to Iya Scambirah Pizza!!
What is the number of people?
""");
int numberofpeople=input.nextInt();
input.nextLine();
System.out.println("""
What Pizza Type would you like Sir/Ma
Sapa Size.
Small Money.
Big Boys.
Odogwu.
""");
String typeofpizza=input.nextLine();
int pizzaslices=0;
int pizzaprices=0;

if(typeofpizza.equalsIgnoreCase("Sapa Size")){
pizzaslices=sapaslice;
pizzaprices=sapaprice;
}
else if(typeofpizza.equalsIgnoreCase("Small Money")){
pizzaslices=smallslice;
pizzaprices=smallprice;
}
else if(typeofpizza.equalsIgnoreCase("Big Boys")){
pizzaslices=bigslice;
pizzaprices=bigprice;
}
else if (typeofpizza.equalsIgnoreCase("Odogwu")){
pizzaslices=odogwuslice;
pizzaprices=odogwuprice;
}
int boxes=1;

while(boxes * pizzaslices < numberofpeople){
boxes++;
}
int bill=boxes * pizzaprices;
int leftoverpizza=boxes * pizzaslices - numberofpeople;


System.out.println("Number of boxes of pizza to buy =" + boxes + "boxes");
System.out.println("Number leftover slice after serving=" + leftoverpizza + "slices");
System.out.println("Prices=" + bill);

}

}

















